source/runtime.cpp

```diff
@@ -4512,6 +4530,11 @@ void reshade::runtime::save_texture(const texture &tex)
 				case 2:
 					save_success = stbi_write_jpg_to_func(write_callback, file, width, height, 4, pixels.data(), _screenshot_jpeg_quality) != 0;
 					break;
+				case 3:
+					if (std::vector<uint8_t> encoded_data;
+						simple_jxl::writer(pixels, encoded_data, _width, _height, 4))
```

Comment:
This is using the wrong width and height, should be `width` and `height`, rather than `_width` and `_height`.